less/bottom-content.less

```diff
@@ -0,0 +1,267 @@
+.home .bottom-content {  
+    padding-bottom: 10%;
+    padding-top: 10%;
+
+    @media @phone {
+      margin: 0 -8%;
+    }
+  
+    h1 {
+      .sub-header();
+      
+      
+
+      @media @phone {
+        padding-left: 7%;
+      }
+    }
+    
+    .location-box {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start; 
+      padding-top: 10%; 
```

Comment:
Not sure what happened since you demoed this for the group, but the location boxes are not at the bottom of the image anymore, but closer to the middle.